### DISCUSSION TITLE
How to detect if Chromecast is still connected to SharpCaster?

### DISCUSSION THREAD

**Discussion by Kanuan** [Tapanila/SharpCaster#239] [Q&A] [answered]
First, thanks for SharpCaster!

I wanted to know how to detect if the Chromecast was turned off, timed out or switched to another App.

From what I understood, the ChromecastClient should dispose itself with the `ChromecastClient.DisconnectAsync()` method when it detects the chromecast has timed out/switched app/turned off. I'm registering for its event like so:

![image](https://github.com/Tapanila/SharpCaster/assets/24910442/3eba89d6-adf2-4c92-abed-ed8ff73fdaf3)

But the event is never triggered unless I manually inoke `DisconnectAsyn()`. Should I be manually detecting (somehow) if the connection is still alive or is it related to the #226 ?

**Comment by Tapanila** [comment, marked as answer]
This was a good catch. Now there's a release of 1.1.1 which handles this scenario.
